erpnext/accounts/doctype/sales_invoice/pos.py

```diff
@@ -378,8 +379,8 @@ def add_customer(data):
 	customer_doc.customer_name = data.get('full_name') or data.get('customer')
 	customer_doc.customer_pos_id = data.get('customer_pos_id')
 	customer_doc.customer_type = 'Company'
-	customer_doc.customer_group = frappe.db.get_single_value('Selling Settings', 'customer_group')
-	customer_doc.territory = frappe.db.get_single_value('Selling Settings', 'territory')
+	customer_doc.customer_group = frappe.db.get_single_value('Selling Settings', 'customer_group') or _("Government")
```

Comment:
What if government does not exists?

I think we should force user to enter default customer group and territory in selling settings, even better if it is specified in POS Profile.